### ISSUE TITLE
Error Installing via CommandBox

### ISSUE THREAD

**Issue by bshea5** [cfsimplicity/lucee-spreadsheet#220]
Get the following when running 
>  box install cfsimplicity/lucee-spreadsheet

× | Installing package [github:cfsimplicity/lucee-spreadsheet]
   | > Error cloning github repository
   |-------------------------------------------------------------------------
   | Cloning Git URL [https://github.com/cfsimplicity/lucee-spreadsheet.git]
   | remote: Enumerating objects
   | remote: Counting objects (339)
   | remote: Compressing objects (225)
   | Receiving objects (3013)
   | Resolving deltas (2135)
   | Checking out files (108)
   |-------------------------------------------------------------------------

ERROR (5.2.1+00295)

Error cloning github repository

org.eclipse.jgit.api.errors.RefNotFoundException: Ref master cannot be resolved

**Comment by cfsimplicity** [comment]
I'm getting the same error. It seems that CommandBox is looking for something in the "master" branch of the repo, but I recently renamed that branch to "main". The current release and `box.json` does reflect that change though. I'm not a CommandBox expert though so I've probably missed something.

I've found that it does work via Forgebox, so try this simpler command instead:
```
box install lucee-spreadsheet
```
